packages/platforms/celestica/x86-64/x86-64-cel-seastone/onlp/PKG.yml

```diff
@@ -0,0 +1,15 @@
+variables:
+  platform: x86-64-cel_seastone-r0
```

Comment:
ONL platform identifiers do not have underscores in them. This should be x86-64-cel-seastone-r0